client/src/app/menu/language-chooser.component.ts

```diff
@@ -28,4 +28,10 @@ export class LanguageChooserComponent {
     return window.location.origin + '/' + lang.id
   }
 
+  getCurrentLanguage () {
```

Comment:
It won't work for every cases. Inject `LOCALE_ID` instead (check in `plugin.service.ts` to see how we use it)